lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1681,12 +1682,14 @@ def generate_certificate(self, url, mode):
         return generated_certificate
 
     def mock_certificate_downloadable_status(
-            self, is_downloadable=False, is_generating=False, is_unverified=False, uuid=None, download_url=None
+            self, is_downloadable=False, is_generating=False, is_unavailable=False, is_unverified=False, uuid=None,
+            download_url=None
     ):
         """Dry method to mock certificate downloadable status response."""
         return {
             'is_downloadable': is_downloadable,
             'is_generating': is_generating,
+            'is_dateunavailable': is_unavailable,
```

Comment:
Another nit, but may as well match the parameter name in the method signature to the field name in the dict we're returning here, like with the other parameters.